### PR TITLE
Standardize Kotlin Code Formatting

### LINKED ISSUE #879: Standardize Formatting

### PR DESCRIPTION
**Pull request by ddufourboivin-zalando**
fix #879 

- Add `.editorconfig`
- Set indent to 4
- Set continuation indent to 4
- add Gradle task to apply ktlint+editorconfig settings to IntelliJ Project
- add instructions on how to contribute in proper Kotlin format in `server/CONTRIBUTING`

Important differences (the rest being simply applying formatting rules):
- https://github.com/zalando/zally/pull/880/files#diff-a1b080369a1999fce09f94befe66e350R23
- https://github.com/zalando/zally/pull/880/files#diff-2f2755def9d7e586d2ee1c3441575874R1
- https://github.com/zalando/zally/pull/880/files#diff-e865ba0efbdc729440e9a1f7283e3798R1
- https://github.com/zalando/zally/pull/880/files#diff-2af3da569e27d5fad9d8c14613c35a66R130

### CODE REVIEW

File: server/build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ configurations {
 }
 
 def versions = [
-    jacoco: '0.7.8',
+        jacoco: '0.7.8',
```
Comment (roxspring) [inline server/build.gradle]: Perhaps we should try switching to `build.gradle.kts` and benefit from consistent formatting here too!

Comment (ddufourboivin-zalando) [reply server/build.gradle]: Good idea. But my personal experience using KTS with Gradle was not successful :(

**Comment by ddufourboivin-zalando** [comment]
👍

**Comment by ddufourboivin-zalando** [comment]
👍

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by ddufourboivin-zalando** [comment]
👍

**Pull request merged**